packages/trader/src/AppV2/Components/TradeParameters/Stake/stake.tsx

```diff
@@ -146,12 +146,7 @@ const Stake = observer(({ is_minimized }: TStakeProps) => {
 
     const handleOnChange = (e: { target: { name: string; value: string } }) => {
         setShouldShowError(e.target.value !== '');
-        // TODO: this value modifications will no longer be needed after quill-ui TextFieldWithSteppers is improved to accept and replace commas with dots:
-        let value = e.target.value.replaceAll(',', '.').replaceAll(/\.{2,}/g, '.');
-        if (Number(value.match(/\./g)?.length) > 1) {
-            value = parseFloat(value).toString();
-        }
-        onChange({ target: { name: 'amount', value } });
```

Comment:
this fixes a console error when using + - on Stake input because value replacement logic is now handled in the latest quill-ui versions.